packages/services/src/processDPP.service.ts

```diff
@@ -39,7 +39,7 @@ export const processDPP: IService = async (data: any, context: IContext): Promis
     });
 
     const storageContext = context.storage;
-    const vcUrl = await getStorageServiceLink(storageContext, vc, `${identifier}/${generateUUID()}`);
+    const vcUrl = await getStorageServiceLink(storageContext, vc, `${identifier}/${qualifierPath}`);
```

Comment:
When we store the `${identifier}/${qualifierPath}` as filename, this means we can override the contain of this credential, is it correct?

Reply:
Yes, the identified path should be the identification of the VC, adding the UUID made it hard to identify the stored file.